backend/internal/testutils/testutils.go

```diff
@@ -0,0 +1,24 @@
+package testutils
+
+import (
+	"encoding/json"
+	"github.com/stretchr/testify/require"
+	"io/ioutil"
+	"net/http"
+	"testing"
+)
+
+// Sends http request, converts data from json, and stores to response
+func SendHttpRequest(req *http.Request, response interface{}, t *testing.T) {
```

Comment:
IMO crafting request objects is a pain in the ass. It would be awesome if we could write most tests in like 2-3 lines.

maybe `func MakeRequest(t *testing.T, ts httptest.Server, method string, url string, response interface{})`?

Reply:
Also, then you can concatenate `ts.URL` with `url` within the function.